plugins/aws_ec2/icon_aws_ec2/actions/authorize_security_group_egress/action.py

```diff
@@ -1,16 +1,15 @@
-import komand
 from .schema import AuthorizeSecurityGroupEgressInput, AuthorizeSecurityGroupEgressOutput
```

Comment:
For each action, please add

```suggestion
from .schema import AuthorizeSecurityGroupEgressInput, AuthorizeSecurityGroupEgressOutput, Component
```
...

Reply:
Thanks, good catch 👍